ephios/core/services/notifications/types.py

```diff
@@ -562,17 +567,15 @@ def send(cls, event: Event, content: str):
                         "email": responsible.email,
                         "event_id": event.id,
                         "content": content,
-                        "event_title": event.title,
                     },
                 )
             )
         Notification.objects.bulk_create(notifications)
 
     @classmethod
     def get_subject(cls, notification):
-        return _("Information for your participation at {title}").format(
-            title=notification.data.get("event_title")
-        )
+        event = Event.objects.get(pk=notification.data.get("event_id"))
+        return _("Information for your participation at {title}").format(title=event.title)
```

Comment:
I think many of our subject lines are too long...

```suggestion
        return _("Information regarding {title}").format(title=event.title)
```